bluetail/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.2.10 on 2020-06-10 21:47
+# Generated by Django 2.2.10 on 2020-06-22 17:53
```

Comment:
What's the reasoning for continually updating the initial migration, rather than creating new ones? I don't think it's necessarily wrong, just wanted to understand the rationale 🙂

Reply:
@chrismytton Initially Django was complaining about changing some fields (to Null=True I think?) but instead of debugging I just recreated migrations from scratch for speed and simplicity, as we have no deployment with data that needs a stable migration and we are recreating the DB from scratch at the moment. Also there was a lot of experimenting and backtracking early on so I just got into that workflow.
Do you think it useful to steps taken in the experiments and fixes in multiple migrations?

Reply:
@SimKennedy That's fine. I'm happy to carry on as we are for now. If/when we get to a stage where we want to keep the database intact then we can start doing additive migrations at that point.